ftw/servicenavigation/upgrades/20200615183133_plone5_upgrade/cssregistry.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+<object name="portal_css" meta_type="Stylesheets Registry">
+
+    <stylesheet
+        id="++resource++collective.z3cform.datagridfield/datagridfield.css"
+        expression=""
+        />
+
```

Comment:
I'd remove this (ignore CSS from that product)

Reply:
Sounds reasonable.